dev-python/ijson/ijson-2.5.1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Iterative JSON parser with a Pythonic interface"
+HOMEPAGE="https://github.com/ICRAR/ijson https://pypi.org/project/ijson/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-libs/yajl"
+DEPEND="${RDEPEND}"
+BDEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+
+# pypi release lacks tests and git release isn't tagged yet
```

Comment:
It's tagged now.

Reply:
Well I agree that it is tagged, but the tests are excluded from the `pypi` release - checked now.
Maybe update the comment, but I'm not sure to what.
I did manually run on the `github` tag release, so maybe switch to it?

Reply:
Yes, just use github snapshot.

Reply:
Updated to use `github` release, just to be sure ran `ebuild ... test` for `python2_7` and `python3_7` - Successful for all 57 tests.